### PR TITLE
mu4e: fix reply forward to msg from headers-view

### LINKED ISSUE #12: `org-msg-post-setup` chokes in  mu4e if message is not currently open
`org-msg-post-setup` calls `org-msg-save-article-for-reply-mu4e` in mu4e mode, which assumes that the buffer `*mu4e-view*` is currently open. However, that may not be the case.  If it's not, the method we currently have for inserting message contents will fail.  

because `org-msg-post-setup` is called at the end of `mu4e-compose` -- the post-execution hook -- it's a bit late at that point to o something simple like just call `mu4e-headers-view-message`. There's a long sequnce of bouncing function calls in mu4e's window-drawing call stack, but it might be possible to extract the `docid` attribute and use it to track down the html body. Maybe it's worth asking on`mu-discuss`?

### PR DESCRIPTION
**Pull request by gspe**
This fix #12 the error that happens when in mu4e try to reply or forward message
from headers view.

### CODE REVIEW

File: org-msg.el [hunk 1]
```diff
@@ -863,8 +862,7 @@ a html mime part, it returns t, nil otherwise."
 
 (defun org-msg-article-htmlp-mu4e ()
   "Return t if the current mu4e article is HTML article."
-  (with-current-buffer mu4e~view-buffer-name
-    (when (mu4e-message-field-at-point :body-html) t)))
+    (when (mu4e-message-field mu4e-compose-parent-message :body-html) t))
```
Comment (jeremy-compostella) [inline org-msg.el:865]: It looks like the indentation is not right.

File: org-msg.el [hunk 2]
```diff
@@ -375,7 +374,7 @@ file."
 			       "")
 		    "</div>\n<hr>\n"))
 	  (write-file file))
-	(list file)))))
+	(list file))))
```
Comment (jeremy-compostella) [inline org-msg.el:377]: I would expect the whole indentation of the function to change.

File: org-msg.el (hunk 1 above)
Comment (gspe) [reply org-msg.el]: Now should be ok

File: org-msg.el (hunk 2 above)
Comment (gspe) [reply org-msg.el]: Now should be ok

**Pull request merged**